### PR TITLE
net-dns/dnstop: Fix BDEPEND, need pkgconfig

### PR DESCRIPTION
**Pull request by gjaekel**
Needs `pgkconfig` to build.

Signed-off-by: Guido Jäkel <G.Jaekel@DNB.DE>

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-dns/dnstop/dnstop-20140915-r4.ebuild [hunk 1]
```diff
@@ -13,9 +13,15 @@ LICENSE="BSD"
...
-	net-libs/libpcap"
+RDEPEND="
+	sys-libs/ncurses:0
+	net-libs/libpcap
+"
 DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+"
+
```
Comment (Flowdalic) [inline net-dns/dnstop/dnstop-20140915-r4.ebuild:24]: nit: extra newline, please drop

Comment (gjaekel) [reply net-dns/dnstop/dnstop-20140915-r4.ebuild:24]: remove L24 completely?

**Comment by Flowdalic** [comment]
Thanks, squash into a single commit please.

**Comment by gjaekel** [comment]
Sorry, I don't find instructions for Guthub Web-UI to accomplish that. :(

**Comment by Flowdalic** [comment]
> Sorry, I don't find instructions for Github Web-UI to accomplish that. :(

Then I'll do it for you this time. But I would suggest to get familiar with git for the next contribution, as not everything can be done via github's web UI.

**Pull request closed without merging**